lib/autobuild/packages/cmake.rb

```diff
@@ -414,12 +419,12 @@ def configure
                         command << "-G#{generator}"
                     end
                     command << srcdir
-                    
+
                     progress_start "configuring CMake for %s", :done_message => "configured CMake for %s" do
                         if full_reconfigures?
                             FileUtils.rm_f cmake_cache
                         end
-                        run('configure', *command)
+                        run('configure', *command, {env: cmake_env})
```

Comment:
One does not explicitly use `{}` for keyword-like arguments. Just `run('configure', *command, env: cmake_env)` would look a lot more like the ruby common style.